salt/modules/network.py

```diff
@@ -528,7 +528,10 @@ def netstat():
 
 def active_tcp():
     '''
-    Return a dict containing information on all of the running TCP connections
+    Return a dict containing information on all of the running TCP connections (currently linux and solaris only)
+
+    .. versionchanged:: Boron
+        Added support for SunOS
```

Comment:
@sjorge, I think we should backport this to 2015.8, so this should be `.. versionchanged:: 2015.8.4` if you don't mind.